portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase

```diff
@@ -428,6 +428,56 @@ definition {
 		ACDXPSettings.connectDXPtoAnalyticsCloud();
 	}
 
+	@description = "Bug: LRAC-10629 | Automation ID: LRAC-11443 | Test Summary: Sites and Contacts connection status is not changing when renaming the data source name"
+	@priority = "3"
+	test ConnectionStatusNotChangeWhenRenameDataSourceName {
```

Comment:
Hey @manuferreira, I think you forgot to do these steps in the test:

1. Disconnected the data source;
2. Observe the Current Status is DISCONNECTED;
3. Rename the Data Source name;
4. Observe the Current Status is DISCONNECTED;
5. Refresh/reload the page;
6. Observe the Current Status is DISCONNECTED;